samples/subsys/logging/dictionary/sample.yaml

```diff
@@ -5,3 +5,16 @@ tests:
   sample.logger.basic.dictionary:
     build_only: true
     tags: logging
+  sample.logger.basic.dictionary.fpu:
+    build_only: true
+    tags: logging
+    filter: CONFIG_CPU_HAS_FPU
+    extra_configs:
+      - CONFIG_FPU=y
+  sample.logger.basic.dictionary.fpu.long_double:
+    build_only: true
```

Comment:
why build only?

Reply:
AFAIK, there is no twister harness to grab the input, feed it to another program, and then compare the result. Since we can't test whether output is correct, there is no point to run it (yet).

Reply:
can we use integration_platforms here to limit amount of times we build this for every PR, right now it is 40 builds :(

Reply:
Added x86 QEMU and qemu_cortex_a53*.